docs/source/reservations/existing-clusters.rst

```diff
@@ -0,0 +1,143 @@
+------------------
+
+1. Create a file ``ips.txt`` with the IP addresses of your machines with one IP per line.
+   The first node will be used as the head node - this node must have port 6443 accessible from your local machine.
+
+   Here is an example ``ips.txt`` file:
+
+   .. code-block:: text
+
+      192.168.1.1
+      192.168.1.2
+      192.168.1.3
+
+   In this example, the first node (``192.168.1.1``) has port 6443 open and will be used as the head node.
+
+2. Run ``sky local up`` and pass the ``ips.txt`` file, SSH username, and SSH keys as arguments:
+
+   .. code-block:: bash
+
+      IP_FILE=ips.txt
+      SSH_USERNAME=username
+      SSH_KEY=path/to/ssh/key
+      sky local up --ip $IP_FILE --username $SSH_USERNAME --key-path $SSH_KEY
+
+   SkyPilot will deploy a Kubernetes cluster on the remote machines, setup GPU support, configure Kubernetes credentials on your local machine, and set up SkyPilot to operate with the new cluster.
+
+   Example output of ``sky local up``:
+
+   .. code-block:: console
+
+      $ sky local up --ips ips.txt --username gcpuser --key-path ~/.ssh/id_rsa
+      Found existing kube config. It will be backed up to ~/.kube/config.bak.
+      To view detailed progress: tail -n100 -f ~/sky_logs/sky-2024-09-23-18-53-14-165534/local_up.log
+      ✔ K3s successfully deployed on head node.
+      ✔ K3s successfully deployed on worker node.
+      ✔ kubectl configured for the remote cluster.
+      ✔ Remote k3s is running.
+      ✔ Nvidia GPU Operator installed successfully.
+      Cluster deployment done. You can now run tasks on this cluster.
+      E.g., run a task with: sky launch --cloud kubernetes -- echo hello world.
+      🎉 Remote cluster deployed successfully.
+
+
+4. To verify that the cluster is running, run:
+
+   .. code-block:: bash
+
+      sky check kubernetes
+
+   You can now use SkyPilot to launch your :ref:`development clusters <dev-cluster>` and :ref:`training jobs <ai-training>` on your own infrastructure.
+
+   .. code-block:: console
+
+      $ sky show-gpus --cloud kubernetes
+      Kubernetes GPUs
+      GPU   QTY_PER_NODE  TOTAL_GPUS  TOTAL_FREE_GPUS
+      L4    1, 2, 4       12          12
+      H100  1, 2, 4, 8    16          16
+      Kubernetes per node GPU availability
+      NODE_NAME                  GPU_NAME  TOTAL_GPUS  FREE_GPUS
+      my-cluster-0               L4        4           4
+      my-cluster-1               L4        4           4
+      my-cluster-2               L4        2           2
+      my-cluster-3               L4        2           2
+      my-cluster-4               H100      8           8
+      my-cluster-5               H100      8           8
+
+      $ sky launch --cloud kubernetes --gpus H100:1 -- nvidia-smi
+
+   .. tip::
+
+     You can also use ``kubectl`` to interact and perform administrative operations on the cluster.
+
+Cleanup
+-------
+
+To clean up all state created by SkyPilot on your cluster, use the ``--cleanup`` flag:
```

Comment:
Reading "all state" made me wonder what are those states? How about adding a quick section with a bullet list / snippet showing this installed state? Users have asked "what do you install on my cluster".